mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -3285,4 +3287,50 @@ resource "google_container_node_pool" "with_tpu_topology" {
 }
 `, cluster, np1, np2, tpuTopology)
 }
-<% end -%>
\ No newline at end of file
+
+func TestAccContainerNodePool_withHostMaintenancePolicy(t *testing.T) {
+	t.Parallel()
+
+	cluster := fmt.Sprintf("tf-test-cluster-%s", RandString(t, 10))
+	np := fmt.Sprintf("tf-test-np-%s", RandString(t, 10))
+
+	VcrTest(t, resource.TestCase{
+			PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+			ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+			CheckDestroy:             testAccCheckContainerNodePoolDestroyProducer(t),
+			Steps: []resource.TestStep{
+					{
+							Config: testAccContainerNodePool_withHostMaintenancePolicy(cluster, np),
+					},
+					{
+							ResourceName:      "google_container_node_pool.np",
+							ImportState:       true,
+							ImportStateVerify: true,
+					},
+			},
+	})
+}
+
+func testAccContainerNodePool_withHostMaintenancePolicy(cluster, np string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+}
+
+resource "google_container_node_pool" "np" {
+  name               = "%s"
+  location           = "us-central1-a"
+  cluster            = google_container_cluster.cluster.name
+  initial_node_count = 1
+  node_config {
+	machine_type = "n2-standard-2"
+    host_maintenance_policy {
```

Comment:
Could you fix the indentation here?

Reply:
Done. Interesting, looks like tab is interpreted as 8 spaces in this editor. Swapped to spaces.